src/bootstrap/configure.py

```diff
@@ -74,6 +74,7 @@ def v(*args):
 o("llvm-plugins", "llvm.plugins", "build LLVM with plugin interface")
 o("debug-assertions", "rust.debug-assertions", "build with debugging assertions")
 o("debug-assertions-std", "rust.debug-assertions-std", "build the standard library with debugging assertions")
+o("randomize-layout", "rust.randomize-layout", "randomize struct layout of rustc, libraries and tools")
```

Comment:
nit: I don't think this will be used frequently enough to need its own option. I would rather use `--set rust.randomize-layout true` in CI and keep these top-level options for stuff we think will be useful in `./configure --help`.